tests/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-if (CMAKE_BUILD_TYPE AND OT_COMMISSIONER_COVERAGE AND CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+if (CMAKE_BUILD_TYPE AND OT_COMM_COVERAGE AND CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
```

Comment:
OT_COVERAGE?

Reply:
Should we use `OT_COMM_` as prefix of all CMake options in OT Commissioner? There will be name conflict if a project includes both `openthread` and `OT Commissioner` as a library.

Reply:
I assume these are common flags. Just like DEBUG. User don't have to set OT_DEBUG and OT_COMM_DEBUG. Thoughts?

Reply:
No, this has an assumption on libraries that they using the same options prefix. I believe it is not the usual case for CMake users. It is always for CMake user to explicitly config each dependency independently. Agree?